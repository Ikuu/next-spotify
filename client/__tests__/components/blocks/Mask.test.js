import React from 'react';
import { create } from 'react-test-renderer';
import Mask from '../../../components/blocks/Mask';

test('Mask renders', () => {
  const tree = create(<Mask><img src="src" alt="alt" /></Mask>).toJSON();
  expect(tree).toMatchSnapshot();
});
