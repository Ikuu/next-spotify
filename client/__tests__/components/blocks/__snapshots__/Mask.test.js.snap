// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mask renders 1`] = `
<div
  style={
    Object {
      "borderRadius": "50%",
      "height": "auto",
      "overflow": "hidden",
      "position": "relative",
      "width": "auto",
    }
  }
>
  <img
    alt="alt"
    src="src"
  />
</div>
`;
